invenio_rdm_records/resources/config.py

```diff
@@ -218,6 +218,14 @@ class IIIFResourceConfig(ResourceConfig):
         "image_format": ma.fields.Str(),
     }
 
+    request_read_args = {
+        "dl": ma.fields.Str(),
+    }
+
+    request_headers = {
+        "If-Modified-Since": ma.fields.DateTime(),
+    }
+
     response_handler = {"application/json": ResponseHandler(JSONSerializer())}
 
     supported_formats = {
```

Comment:
I am a bit out of context, sorry... do you know we have `IIIF_FORMATS` config but also this `supported_formats`?
Should they match?

Reply:
You are right, and I missed the `supported_formats`. 
Now I have a question for you 😄 
How can I access the resource config from the service? (I don't think I should 😂 )

Also, and answering your second question, `IIIF_FORMATS` it's used by Flask-IIIF to do its thing. A good alternative might be to remove the `supported_formats` and rely solely on `IIIF_FORMATS` everywhere. 
I can't think of any reason I added it in the first place ...